NVDAVersions.json

```diff
@@ -0,0 +1,26 @@
+{
+	"2019.3": {
```

Comment:
Please add a json schema for this file.

Reply:
I've added this, and changed the schema to work as an array as you suggested yesterday, as I think it makes more sense in this instance.